rpc/src/server.rs

```diff
@@ -100,24 +102,14 @@ impl RpcServer {
         };
         let method_router = post_router.merge(get_router);
 
-        let mut app = Router::new()
+        let app = Router::new()
             .route("/", method_router.clone())
             .route("/*path", method_router)
             .route("/ping", get(ping_handler))
             .layer(Extension(Arc::clone(rpc)))
             .layer(CorsLayer::permissive())
             .layer(TimeoutLayer::new(Duration::from_secs(30)))
-            .layer(Extension(stream_config.clone()));
-
-        if enable_websocket {
-            let ws_config: StreamServerConfig =
-                stream_config
-                    .with_keep_alive(true)
```

Comment:
Is it correct to remove `with_keep_alive(true)` from this PR?

Reply:
there is already a `with_keep_alive` at line 90.